### DISCUSSION TITLE
Resource Tracking Thread for Make.WordPress.org Redesign

### DISCUSSION THREAD

**Discussion by courtneyr-dev** [WordPress/Marketing-Team#387] [Show and tell]
### Comment

This issue is a place to gather resources and research on overhauling Make.WordPress.org's text.

**Comment by courtneyr-dev** [comment]
https://www.emilyomier.com/blog/project-homepages-that-dont-suck

Video: https://ftp.belnet.be/mirror/FOSDEM/video/2024/h2215/fosdem-2024-3154-project-websites-that-don-t-suck.mp4 

Overview:

# FOSDEM 2024: Project Websites That Don't Suck

## Introduction
- **Speaker**: Emily Omier
- **Focus**: Content quality over design aesthetics in project websites.

## Background and Focus
- Emily is a positioning consultant for open source startups.
- Hosts "The Business of Open Source" podcast.

## Homepage Goals
- **Quick Understanding**: Convey what the project does and its value promptly.
- **Self-Selection**: Enable visitors to determine relevance and fit.
- **Credibility Boost**: A well-crafted homepage enhances project credibility.

## Homepage Analytics and Audience Insights
- **Bounce Rates**: Not a reliable indicator of homepage effectiveness.
- **Efficiency**: The goal is to quickly repel unfit visitors to save everyone's time.
- **Navigation**: Clear links to documentation and about pages are crucial for developer-oriented projects.

## Anatomy of a Homepage
- **Essential Elements**:
  - Market category
  - Project benefits
  - Ideal fit clarification
  - Unique value proposition
- **Examples**:
  - Open Telemetry for clarity
  - Apache Cassandra for differentiated value

## Importance of About Pages
- **Common Issues**: Many about pages fail to convey the project's uniqueness or team humanity.
- **Effective About Pages**: Showcase the team, their viewpoints, and build credibility.

## Avoid Basic Mistakes
- **Professionalism**: Correct grammar, punctuation, and consistent capitalization are crucial.
- **Perception**: Errors may reflect poorly on project quality.

## Assessing Your Website
- **Feedback**: Direct feedback from community channels can provide clarity and effectiveness insights.

## Recap and Conclusion
- **Purpose of Websites**: Facilitate efficient self-selection and convey project essence.
- **Key Pages**: Don't neglect the about page.
- **Accuracy**: Ensure basic correctness to maintain credibility.

**Comment by eidolonnight** [comment]
Thanks @courtneyr-dev ! Moving this to the Discussion area until we have actionable items to break off into issues later.
